.github/workflows/release-drafter.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   update_release_draft:
     runs-on: ubuntu-latest
     steps:
-      - uses: release-drafter/release-drafter@fe52e97d262833ae07d05efaf1a239df3f1b5cd4 # pin@v5
+      - uses: release-drafter/release-drafter@v5
         with:
```

Comment:
Do we want to keep this? Github now allows you to [generate release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) based on PRs merged, so we should be all set.

Reply:
i would like to remove it, if no one is against then ill just drop it.